db.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+
+import click
+from flask import current_app, g
+from flask.cli import with_appcontext
+
+
+def get_db():
+    if 'db' not in g:
+        g.db = sqlite3.connect(
+            'coffeemaker.sqlite',
+            detect_types=sqlite3.PARSE_DECLTYPES
```

Comment:
this is ok, but you should add methods for CRUD operations as well. just call those instead of manipulating raw sql queries